taxcalc/tests/test_calculate.py

```diff
@@ -45,38 +44,19 @@ def paramsfile():
     os.remove(f.name)
 
 
-def add_df(alldfs, df):
-    for col in df.columns:
-        if col not in all_cols:
-            all_cols.add(col)
-            alldfs.append(df[col])
-
-
-def run(puf=True):
-
-    # Create a Parameters object
-
-    params = Parameters(start_year=1991, inflation_rates=irates)
-
-    # Create a Public Use File object
-    puf = Records(tax_dta)
-
-    # Create a Calculator
-    calc = Calculator(params=params, records=puf)
+def run():
+    parm = Parameters()
+    assert parm.current_year == 2013
+    recs = Records(data=TAX_DTA, weights=WEIGHTS, start_year=2009)
```

Comment:
So implicit in this PR, we are saying that it is OK to override what is written in the underlying data when creating a Records instance by passing a `start_year` keyword arg, but it is not OK to do that for the Parameters instance. For example, the tax record data is from 1991, but we create the Records instance for 2009. However, we are not allowed to create a Parameters instance earlier in time, even if we wanted to. I will be using this interface far less often than others, so I don't have particularly strong feelings, it just feels a bit arbitrary and should probably be written down somewhere.